packages/teams-js/src/public/stageView.ts

```diff
@@ -42,6 +42,31 @@ export namespace stageView {
      * The entity ID of the Teams application content being opened.
      */
     entityId?: string;
+
+    /**
+     * The open mode for stage content.
+     * Optional param: if not passed, hosts can choose their default openMode.
+     * If a host does not support any specific openMode, they can fallback to StageViewOpenMode.modal.
```

Comment:
I'd like this to be more proscriptive; it is hard for developers to deal with language like "can fallback" because it doesn't make it easy for them to rely on what the behavior will actually be across different hosts. I think what we agreed to is that hosts *have to* fall back to modal if they can't open one the requested mode.

Reply:
```suggestion
     * If a host does not support any specific openMode, they will fallback to StageViewOpenMode.modal.
```

Reply:
Yes. I have fixed it now.